freqtrade/rpc/api_server/ws/schema.py

```diff
@@ -0,0 +1,66 @@
+from datetime import datetime
+from typing import Any, Dict, List, Optional
+
+from pandas import DataFrame
+from pydantic import BaseModel, ValidationError
+
+from freqtrade.constants import PairWithTimeframe
+from freqtrade.enums.rpcmessagetype import RPCMessageType, RPCRequestType
+
+
+__all__ = ('WSRequestSchema', 'WSMessageSchema', 'ValidationError')
```

Comment:
I don't think  that's necessary - we're not using `from ...schema import *` - and this is not useful in other scenarios afaik.

it's not even protecting members not listed here as they can still be imported by name.

we should also rename this file to `ws_schemas.py` - that'd make it aligned to `api_schemas.py`